pkg/network/dhcp/server/server.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-package dhcp
+package server
```

Comment:
Why there is a need for a separated package? Could you please explain?

Reply:
First approach was copy it under ./pkg/network/dhcp but problem is that this package depends on the ./pkg/network/driver and that package depend on the dhcp server (the driver package containe the NetworkHandler interface with the StartDHCPServer function) so it creates a cycle, moving it to it's own package fix the issue.

Reply:
I see. It seems an artificial fix to a problem which should solved from the roots.
But this is a good step forward.

We should break down the driver package.

Reply:
It's ok to do that as a follow up ?

Reply:
Yes, I didn’t meant to fix it now.